### DISCUSSION TITLE
Couldn't open a local copy of cd-rom

### DISCUSSION THREAD

**Discussion by carvalhar** [bottlesdevs/Bottles#296] [General]
Hi,

I have Bottles working fine, but i couldn't run the program from this cd rom:
https://archive.org/details/gi2e_20200319

I could install the software, and it appears in control panel as installed, but it does'nt show at the "Programas" tab and I can't open it. Doesn't appear any error.

What should I do?

thanks

**Comment by mirkobrombin** [comment]
Not all programs appear in the program list due to how individual developers' installers are handled.What you can try to do is look for it in the drive_c directory and start it manually.

**Reply by carvalhar** [reply]
It gives me an error message. I attached the printscreen.
![Captura de tela de 2021-05-31 16-30-58](https://user-images.githubusercontent.com/20246049/120234087-9c81c800-c22d-11eb-9f2d-6b7dc6af921e.png)

**Reply by mirkobrombin** [reply]
Here too, as in your other issue, it seems more of a problem with Wine than with Bottles
